scripts/run-cypress-tests.sh

```diff
@@ -40,6 +40,7 @@ echo "Running tests with GDB: $GDB_VERSION"
 DOWNLOADS=.downloads
 DOWNLOAD_URL="http://maven.ontotext.com/repository/owlim-releases/com/ontotext/graphdb/graphdb-free/${GDB_VERSION}/graphdb-free-${GDB_VERSION}-dist.zip"
 GDB_ZIP="$DOWNLOADS/graphdb-$GDB_VERSION.zip"
+export JAVA_HOME="/opt/jdk8u232-b09/jre"
```

Comment:
Is this really needed? This will force to update it (the script) everytime Java is also updated.

Reply:
Oh, no, thanks, actually this whole script is not used anymore.